rust-arroyo/src/backends/kafka/types.rs

```diff
@@ -1,6 +1,7 @@
 use crate::types::TopicOrPartition;
 use rdkafka::message::{BorrowedHeaders, Header, OwnedHeaders};
 use rdkafka::producer::BaseRecord;
+use rdkafka::message::Headers as HeaderTrat;
```

Comment:
```suggestion
use rdkafka::message::Headers as _;
```

If you do not want to *name* traits directly, you can import them as `_` to avoid naming conflicts.